### PR TITLE
Update data.toml to add more responses

### PR DESCRIPTION
**Pull request by Rairof**

### CODE REVIEW

File: src/data.toml [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
 enable_experiments = "Head to Settings > Bunny > Activate Discord Experiments"
 account_switcher = "Enable Discord Experiments and look for the 'Mobile Account Switcher' experiment and set it to Treatment 1. Hold your profile in the navigation tab to manage your accounts."
 disable_nmu = "The only way to disable NMU is to use Discord version <= 224 and change the experiment's treatment. Keep in mind that support is only provided for the latest stable release and your version may stop working at some point."
-fix_voice_activity = "The cause of this is unclear. To fix it, please disable 'Advanced Voice Activity' in voice settings."
+fix_voice_activity = "### App doesn't register your voice in VC:\n\n- Check if your mic is on and all permissions are granted to the app.\n- The cause of this is unclear. To fix it, please disable `Advanced Voice Activity` in voice settings.\n- Disable `Auto Sensitivity` & `Automatic Gain Control` located in `settings/Voice` if discord still doesn't register your voice."
```
Comment (castdrian) [inline src/data.toml:8]: this seems like it was fine before, disabling sensitivity and gain doesn't fix anything more than what Advanced Voice Activity does

File: src/data.toml [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
 custom_fonts = "Fonts are experimental and some packs may break the application. You can get font packs listed here: https://discord.com/channels/1196075698301968455/1225584192012947506/1241453802184380418"
 ios_notifications = "For notifications to work on a Jailed device you need to sign the ipa with a certificate that has the notification entitlement enabled.\n\nPlease keep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
-thirty_two_bit_android_os = "32-bit versions of Android OS are not supported. Please use a 64-bit operating system."
+thirty_two_bit_android_os = "32-bit versions of Android OS are not supported. Please use a 64-bit operating system.\n### Use a System Info App:\nDownload a system information app from the Google Play Store, like "CPU-Z" or "AIDA64." \nThese apps provide detailed information about your device's hardware, including the CPU architecture. \n### Terminal Emulator (Rooted Devices):If your device is rooted, you can use a terminal emulator."
```
Comment (castdrian) [inline src/data.toml:12]: this is malformed toml syntax

File: src/data.toml [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
 no_passkeys_when_pre_patched = "Passkeys are not supported when using the pre-patched application. Please use a different method to log in to your account or use the LSPosed module / iOS tweak."
 ios_app_icons = "When sideloading with a paid certificate the application will be signed using a wildcard App ID.\nThis effectively breaks being able to change app icons.\nYou need to sign the application with a local dev cert instead if you want this functionality.\n\nPlease keep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
-addon_404 = "You are not supposed to visit the url directly since it does not serve anything that can be rendered. Use it to install the add-on."
+addon_404 = "You are NOT supposed to visit the url directly since it does not serve anything that can be rendered. Copy the link and use it to install the add-on."
+not_support_channel = "### You are using the wrong channel!\nThis is not the right channel to ask for any support or requests, please consider going in https://discord.com/channels/1196075698301968455/1210652540757090315 for those."
```
Comment (castdrian) [inline src/data.toml:18]: this should probably be formulated a bit more grammatically eloquent
```suggestion
not_support_channel = "This is not the support channel, please move to https://discord.com/channels/1196075698301968455/1210652540757090315."
```

File: src/data.toml [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
+get_addons = "Bunny offers compatibility with Vendetta (discontinued) & Revenge's(Discontinued) plugins and themes, since Vendetta is what Bunny is based on:\n\n### For plugins:\n- You can retrieve plugin links from: https://bn-plugins.github.io/vd-web/\n- Alternatively you can kindly, request for other plugins that you may spot others using in the server that aren't available in the plugin web list.\n### For themes:\n\n- You may install any Vendetta themes you like, preferably more recent ones are recommended as older themes are likely outdated.\n- Revenge themes are also supported, excluding custom fonts support.\n- You can install themes from Enmity as well but theme's colours may be different  depending on your device platforms considering discord uses different strings for both android & iOS.\n\nYou may find a curated list of broken plugins in https://discord.com/channels/1196075698301968455/1210652540757090315/1229376326528663596"
```
Comment (castdrian) [inline src/data.toml]: this has inconsistent spacing and casing in various positions, and perhaps is a bit much to read

Comment (Rairof) [reply src/data.toml]: got it

Comment (Rairof) [reply src/data.toml]: I will summarise it in a condensed form then

File: src/data.toml (hunk 2 above)
Comment (Rairof) [reply src/data.toml:12]: > this is malformed toml syntax

My bad it slipped my view. Fixed it.

File: src/data.toml (hunk 4 above)
Comment (Rairof) [reply src/data.toml]: done

**Comment by Rairof** [comment]
Kindly review the new committed changes

**Pull request merged**